Automattic/pocket-casts-ios
# Playback 2023: disable it

This PR should be merged on the first week of January 2024 so 7.54 is released without Playback 2023.

I also removed some outdated 2022 code and images.

## To test

1. Run the app
2. Go to Profile
3. ✅ You should not see the Playback 2023 banner there

## Checklist
